alacritty.yml

```diff
@@ -205,6 +205,9 @@ key_bindings:
 mouse_bindings:
   - { mouse: Middle, action: PasteSelection }
 
+multiclick_threshold_ms: 300
```

Comment:
That sounds reasonable -- I can jump on it as soon as I have time (which will probably be sometime **tomorrow** afternoon). If you would like to manually jump in and apply the changes yourself to close this PR sooner, I am fine with that too.